pages/common/xxd.md

```diff
@@ -0,0 +1,23 @@
+# xxd
```

Comment:
_That's_ what it's called! I've been looking for this command. I think I'm going to have to alias it...!